site/_js/analytics.js

```diff
@@ -47,7 +47,14 @@ function sendToGoogleAnalytics({name, delta, id, attribution, navigationType}) {
 
     // See: https://web.dev/debug-web-vitals-in-the-field/
     [dimensions.WEB_VITALS_DEBUG]: webVitalInfo,
-    [dimensions.NAVIGATION_TYPE]: navigationType,
+    // Override for 'navigational-prefetch' for the prefetch origin trial
+    // experiment (https://github.com/GoogleChrome/web.dev/pull/9532)
```

Comment:
Excuse me being annoying, technically this should be https://github.com/GoogleChrome/developer.chrome.com/pull/5216. 😄